content/07.pathogenesis.md

```diff
@@ -96,6 +96,19 @@ While these may be valuable resources, it is important to note that the estimate
 - Antibody production against SARS-CoV-2 by patient who recovered vs patient who did not recover
 - Cytokines and other soluble factors contribution to immune response
 
+Pathogens infect hosts and are fought by host immune defenses.
+However, pathogens have developed ways to evade the immune system as well as various ways to manipulate the host immunity.
+Other general complexities to consider include the following: disease symptoms can be due to a pathogen or due to the reaction of the immune system to the pathogen; the host immune system can defend against or fight infection but it can also sometimes cause too much damage to the host [@isbn: 9780815332183; @isbn: 9780071283663].
```

Comment:
```suggestion
Other general complexities to consider include the following: disease symptoms can be due to a pathogen or due to the reaction of the immune system to the pathogen; the host immune system can defend against infection but it can also sometimes cause too much damage to the host [@isbn: 9780815332183; @isbn: 9780071283663].
```

Reply:
based on feedback/edits from Dr. Nadia Danilova.